src/core/lib/channel/promise_based_filter.h

```diff
@@ -449,6 +572,25 @@ MakeFilterCall(Derived* derived) {
 template <typename Derived>
 class ImplementChannelFilter : public ChannelFilter {
  public:
+  void InitCall(CallSpineInterface* call_spine) {
```

Comment:
Please add a comment explaining that this is doing v3->v2 conversion (ability to run a v3 filter in a v2 stack), and that it transitively works for v3->v1 because we have v2->v1 already implemented.